aip/general/0003.md

```diff
@@ -0,0 +1,34 @@
+---
+id: 3
+state: approved
+created: 2023-03-28
+placement:
+  category: meta
+  order: 30
+---
+
+# AIP Versioning
+
+This AIP defines the versioning scheme of the AIPs.
+
+## Guidance
+
+- The AIPs **must** be versioned by date, using the [ISO-8601 format of `YYYY-MM-DD`](https://www.w3.org/TR/NOTE-datetime), that corresponds to the date the version was added.
+- AIPs versions **must** be available as a tag on the source control mechanism used to store the AIPs, of the format `v{version}`. Example: `v2023-03-28`.
```

Comment:
Nit: s/mechanism/provider/

Reply:
would you mind clarifying why you think that's clearer?

Reply:
Source control is a mechanism, so it is confusing. `provider` or, better yet, `system` differentiate one src control technology from another.

Reply:
I was thinking "technology" was good one too. But I agree system is clearer than mechanism, will fix!